src/Microsoft.TemplateEngine.Edge/Settings/TemplateInfoReaders/TemplateInfoReaderVersion1_0_0_1.cs

```diff
@@ -16,8 +16,6 @@ public static new TemplateInfo FromJObject(JObject jObject)
         public override TemplateInfo Read(JObject jObject)
         {
             TemplateInfo info = base.Read(jObject);
```

Comment:
We could get rid of the local variable and return directly.

Reply:
Or delete whole override? :) Didn't bother commenting since I want to nuke this anyway :)